<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirspaceDialog</class>
 <widget class="QDialog" name="AirspaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Load User Airspaces</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_15">
     <property name="title">
      <string>User Airspaces</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_30">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelAirspacePath">
        <property name="text">
         <string>&amp;User Airspace Base Directory:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditAirspacePath</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonAirspacePathSelect">
        <property name="text">
         <string>&amp;Select ...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditAirspacePath">
        <property name="toolTip">
         <string>Select the directory that contains the user airspace text files.</string>
        </property>
        <property name="placeholderText">
         <string>No directory for user airspaces selected.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditAirspaceExtensions">
        <property name="toolTip">
         <string>Space separated list of file patterns to read as OpenAIR, IVAO JSON or VATSIM GeoJSON airspace files.</string>
        </property>
        <property name="text">
         <string notr="true">*.txt *.json *.geojson</string>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelAirspaceExtensions">
        <property name="toolTip">
         <string>Files with these extensions will be loaded as airspaces.</string>
        </property>
        <property name="text">
         <string>File &amp;extensions:</string>
        </property>
        <property name="buddy">
         <cstring>lineEditAirspaceExtensions</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="labelAirspacePathStatus">
        <property name="text">
         <string>Valid</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditAirspacePath</tabstop>
  <tabstop>pushButtonAirspacePathSelect</tabstop>
  <tabstop>lineEditAirspaceExtensions</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
